src/current/cockroachcloud/serverless-resource-usage.md

```diff
@@ -95,13 +101,64 @@ Time: 71ms total (execution 20ms / network 50ms)
 
 This will insert the data, and also output information from the optimizer about the execution of the statement. The `estimated RUs consumed` field represents the optimizer's estimate of RU consumption for the statement. In this case, the optimizer estimated the `INSERT` statement would consume 15 RUs, which is similar to the estimate of 18.5 RUs we made earlier.
 
-## Understanding which queries to optimize
+## Diagnose and optimize your resource consumption
+
+In the CockroachDB {{ site.data.products.cloud }} Console, you can monitor your cluster's SQL activity on the [**Statements**]({% link cockroachcloud/statements-page.md %}) and [**Transactions**]({% link cockroachcloud/transactions-page.md %}) pages. You can sort queries by the time they took to process, the number of rows processed, or the number of bytes read to see which queries are using the most resources. If you have queries that return more data than needed or have long runtimes, those are good candidates for optimization. 
+
+### Expensive queries
```

Comment:
In this section, we should include the content Shannon pointed out in the comments of the original issue (we do talk about some things he mentioned, but not all of them): 

Expensive queries are the most common culprit, specifically those that involve FULL SCAN operations. Note also that FULL SCAN operations can become increasingly costly as the table size grows.

To diagnose unexpected RU usage, review the SQL Activity page and look for:

Queries that meet two criteria:

1) scanning a significant number of rows

2) processing substantial data

For optimal performance, a well-designed application should strive to limit row scanning to around 50 rows per statement and data scanning to approximately 1K per row, especially for high-frequency queries. Note that occasional queries, like daily reporting tasks, can tolerate larger row and data scanning without causing significant concerns.

We should probably elaborate on what we mean by processing a substantial amount of data. What types of data or queries are likely to exhibit this particular characteristic?